integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/mock/DistroXClusterUpscaleDownscaleTest.java

```diff
@@ -120,6 +120,36 @@ public void testScaleDownAndUpWithLargeNodes(MockedTestContext testContext, ITes
                 .validate();
     }
 
+    @Test(dataProvider = TEST_CONTEXT_WITH_MOCK)
+    @Description(
+            given = "there is a running DistroX cluster with 300 instances in worker group",
+            when = "up- and downscale is called many times",
+            then = "the cluster should be available")
+    public void testScaleDownAndUpManyTimes(MockedTestContext testContext, ITestContext testNgContext) {
+        String stack = resourcePropertyProvider().getName();
+        createDatalake(testContext);
+        DistroXTestDto currentContext = createDistroxDto(testContext, stack, 10)
+                .when(distroXClient.create(), key(stack))
+                .await(STACK_AVAILABLE, key(stack));
+
+        currentContext = currentContext
+                .when(distroXClient.scale(HostGroupType.WORKER.getName(), 10))
+                .await(DistroXTestDto.class, STACK_AVAILABLE, key(stack), POLLING_INTERVAL);
```

Comment:
why do we need this? the test initiates a DH with 10 nodes then this tries to scale worker group to 10 nodes

Reply:
also the method is very similar to `testScaleDownAndUpWithLargeNodes`, only the nodeCount and the number of iteration is different for me

Reply:
Right, thanks for pointing that out. I extracted a common method for the two tests as you suggested, this should be fixed.